http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -913,6 +931,85 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
         else Async[F].pure((mt, true))
       }
     } yield targetMachineType
+
+  private[service] def getAuthorizedIds(
```

Comment:
This code is duplicated from RuntimeV2ServiceInterp. I did not want to complicate the existing codepath by extracting, since the goal is to eventually deprecate the v1 endpoints entirely